reviews/graduation-jaeger.md

```diff
@@ -0,0 +1,156 @@
+    * Quote: "Jaeger is helping us to increase observability since Feb 2018 and is praised by our developers."
+  * [Weave HQ](https://www.getweave.com/) threads together data, software and communication platforms to build stronger relationships at the point of contact. Deployed at over 8000 medical offices. Using Jaeger since early 2018; ~150 active microservices, ~500 containers, ~90% of microservices are instrumented for tracing. Using Cassandra as span storage backend, 1 week retention, ~2M spans/day.
+    * Fun fact 1: started collecting fewer metrics in Prometheus because Jaeger works much better for root cause analysis than having extra metrics.
+    * Fun fact 2: most application logs are written to Jaeger spans via the OpenTracing API and viewed in Jaeger UI in the context of the trace, instead of log aggregation tools like ELK.
+  * [Candide](https://candide.eu/), a mobile app for gardeners, uses Jaeger in production running on GKE with an Elasticsearch backend hosted on Elastic Cloud. 30+ microservices.
+  * [Ticketmaster](https://www.ticketmaster.com/), a ticket sales and distribution company based in Beverly Hills, California, with operations around the world.
+    * Traces 100 million transactions per day with Jaeger .
+    * Has 50-plus services instrumented and nine groups of engineering teams actively using Jaeger.
+    * Quote: "Jaeger Tracing is helping us achieve our vision of observability capabilities across different versions of our platforms, how they integrate with each other and how they have grown over time." -- Kraig Amador, Senior Director.
+  * [Red Hat](https://www.redhat.com/en), world’s leading provider of enterprise open source solutions, using a community-powered approach to deliver high-performing Linux, cloud, container, and Kubernetes technologies.
+    * Jaeger will be fully supported by Red Hat as part of its OpenShift Service Mesh product due for release in a couple of months.
+  * Many other companies self-reporting on https://stackshare.io/jaeger/in-stacks/ajax
+
+Jaeger is also being integrated with other open source projects:
+
+  * OpenCensus libraries and agent ship with [exporters for Jaeger](https://opencensus.io/guides/exporters/supported-exporters/java/jaeger/)
+  * [Istio comes with Jaeger included](https://istio.io/docs/tasks/telemetry/distributed-tracing/)
+  * [Envoy works with Jaeger native client](https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/jaeger_native_tracing)
+  * Eclipse Trace Compass incubator supports [importing traces from Jaeger](https://github.com/tuxology/tracevizlab/tree/master/labs/303-jaeger-opentracing-traces)
+
+## Project long term sustainability
+
+  * Uber has a team of 7 full time engineers working on Jaeger, both internally and in the open source.
+  * Red Hat is planning to include Jaeger as part of its OpenShift Service Mesh product
```

Comment:
Update with current status?

Reply:
@objectiser can you provide some blurb here?

Reply:
As mentioned in a [previous comment](https://github.com/cncf/toc/pull/171#discussion_r320607539) by @jpkrohling, Jaeger is a fully supported part of the recently announced OpenShift Service Mesh product.

Reply:
Thanks, let's just get the text updated in the next push.

Reply:
done